src/bacnet/basic/service/h_ts.c

```diff
@@ -66,6 +66,20 @@ static void show_bacnet_date_time(BACNET_DATE *bdate, BACNET_TIME *btime)
 }
 #endif
 
+static handler_timesync_set_callback_t handler_timesync_set_callback;
+
+static int32_t Time_Offset; /* Time offset in ms */
+
+int32_t handler_timesync_offset()
```

Comment:
Please add 'void' to function parameters. 
Please add JavaDoc style doxygen function header comments to new public API functions.

Reply:
Thanks Steve. Fixed in 898923b9e0f690f3dc63f5ea0c1ee7d8316a041e